### ISSUE TITLE
Heatmap problem VGG

### ISSUE THREAD

**Issue by raphaeli** [heuritech/convnets-keras#15]
Hi,
The heatmap works fine for me for the example provided (see below), but it does not work if you change weights to VGG16 or VGG19. The heatmaps produced by VGG are blank and with strange coordinates.
im = preprocess_image_batch(['examples/dog.jpg'], color_mode="bgr")

sgd = SGD(lr=0.1, decay=1e-6, momentum=0.9, nesterov=True)
model = convnet('alexnet',weights_path="weights/alexnet_weights.h5", heatmap=True)
model.compile(optimizer=sgd, loss='mse')
out = model.predict(im)

s = "n02084071"
ids = synset_to_dfs_ids(s)
heatmap = out[0,ids].sum(axis=0)
plt.imshow(heatmap)
